Jenkinsfile

```diff
@@ -148,7 +148,11 @@ pipeline {
                 stage('daosLatestVersion() tests') {
                     steps {
                         script {
+                            testVar4 = daosLatestVersion('release/2.4', 'el8')
+                            testVar6 = daosLatestVersion('release/2.6', 'el8')
                             assert(daosLatestVersion('master', 'el8').matches(/2.7\.\d+.*/))
+                            echo "daosLatestVersion('release/2.4', 'el8')" + testVar4
+                            echo "daosLatestVersion('release/2.6', 'el8')" + testVar6
```

Comment:
Given the example: https://github.com/daos-stack/pipeline-lib/blob/1623921eebb16ec05a104c97105ed8d9515a9b19/Jenkinsfile#L614-L615 there appears to be an `assert()` signature that takes two arguments, where the first is the test and the second is a string that is displayed when the assertion fails.  This second argument is very useful to put the values being tested (i.e. as a string message) into so that those are displayed with the assertion failure.

It would be good to update these `assert()`s to do this so that this kind of after-the-fact debugging doesn't need to be added just to see why the assertion failed.